refresh/templates/common/Application.swift

```diff
@@ -26,18 +26,19 @@ public var basePath = "<%- basepath %>"
 public let health = Health()
 <% } -%>
 <% if (appInitCode.services.length > 0) { -%>
-
 class ApplicationServices {
-    // Service references
+    // Initialize services
 <% appInitCode.service_variables.forEach(function(variable) { -%>
     <%- variable %>
 <% }); -%>
 
     public init(cloudEnv: CloudEnv) throws {
+<% if (appInitCode.services.length) { -%>
```

Comment:
Isn't this redundant with the check on line 28?

Reply:
fixed.